pact4s/src/main/scala/org/broadinstitute/dede/workbench/leonardo/consumer/FakeClient.scala

```diff
@@ -0,0 +1,68 @@
+package org.broadinstitute.dede.workbench.leonardo.consumer
+
+import cats.effect.kernel.Concurrent
+import cats.implicits.{catsSyntaxApplicativeErrorId, catsSyntaxApplicativeId, catsSyntaxOptionId, toFunctorOps}
+import cats.syntax.all.none
+import io.circe.syntax.EncoderOps
+import io.circe.{Decoder, Encoder, Json}
+import org.http4s.Credentials.Token
+import org.http4s.circe.{jsonEncoderOf, jsonOf}
+import org.http4s.client.Client
+import org.http4s.headers.Authorization
+import org.http4s.{EntityDecoder, EntityEncoder, Method, Request, Status, Uri}
+
+trait FakeClient[F[_]] {
+  def fetchResource(id: String)(implicit decoder: Decoder[Resource]): F[Option[Resource]]
+
+  def createResource(resource: Resource): F[Unit]
+}
+
+/*
+ This class represents the consumer view of a fake provider that implements the following endpoints:
```

Comment:
I kinda prefer just remove this fake client

Reply:
The fake client has been removed.